project/templates/unicoremc/home.html

```diff
@@ -54,14 +57,20 @@
             var projects = new Array();
             $.each(data, function(){
                 var repo_url = this.repo_url.replace(/^(https?):\/\/github.com\//, '');
-                var frontend_url = this.frontend_url.replace(/^(https?):\/\//, '');
-                var cms_url = this.cms_url.replace(/^(https?):\/\//, '');
+                var frontend_url = this.frontend_url.replace(/^(https?):\/\//, '').replace(/\.?(hub\.unicore\.io)/, '');
+                var cms_url = this.cms_url.replace(/^(https?):\/\//, '').replace(/\.?(hub\.unicore\.io)/, '');
+
+                var languages = new Array();
+                for (key in this.available_languages){
+                    languages.push(' <a href="' + this.frontend_url + '/locale/'+ this.available_languages[key] +'/" target="_blank">'+ this.available_languages[key] + '</a>');
+                }
 
                 var row = '<th>' + this.app_type +'</th>' +
                           '<td>' + this.country + '</td>' +
                           '<td>' + this.state + '</td>' +
-                          '<td><a target="_blank" href="'+ this.repo_url +'">'+ repo_url +'</a></td>' +
+                          '<td><a target="_blank" href="'+ this.repo_url +'">Github</a></td>' +
```

Comment:
Yes, that's the only concern I had with the `universalcore/` thing. I'd rather remove it completely since more often than not, you'll just click through to the repo.